app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class UserMailer < ApplicationMailer
+  default from: Rails.application.secrets.from_email
```

Comment:
If you kept the line in ApplicationMailer you should remove it from here.
If you remove it from ApplicationMailer, you then keep this one instead.